app/src/main/java/com/lagradost/cloudstream3/MainAPI.kt

```diff
@@ -163,6 +163,38 @@ object APIHolder {
         return null
     }
 
+    /**
+     * Get anime tracker information based on title, year and type.
+     * Both titles are attempted to be matched with both Romaji and English title.
+     * Uses the consumet api.
+     *
+     * @param mainTitle Title to search by and match
+     * @param secondaryTitle Optional extra title if you have multiple titles and want extra guarantee to match.
+     * @param types Optional parameter to narrow down the scope to Movies, TV, etc. See TrackerType.getTypes()
+     * @param year Optional parameter to only get anime with a specific year
+     **/
+    suspend fun getTracker(
+        mainTitle: String,
```

Comment:
do a List<String> instead here, then smth like matchingTitles = media.title?.let { title -> titles.any { userTitle -> title.isMatchingTitles(userTitle) } } :? true

this is to match any number of titles

Reply:
and uses the first title for search

Reply:
True, true. I forgor :skull:

Reply:
@Blatzar I understand how to do the other change, but don't understand what blob wants in this one. How would I make a list... I sound really dumb rn, I'm tired